compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -28,6 +28,7 @@ use crate::ty::fast_reject::SimplifiedType;
 use crate::ty::util::Discr;
 pub use adt::*;
 pub use assoc::*;
+use ast::expand::StrippedCfgItem;
```

Comment:
```suggestion
use rustc_ast::expand::StrippedCfgItem;
```
Editor auto importing strikes again.